projects/plugins/crm/includes/ZeroBSCRM.DAL3.Obj.Invoices.php

```diff
@@ -1900,6 +1908,7 @@ public function addUpdateInvoice($args=array()){
                             'extraMeta'=>$confirmedExtraMeta #} This is the "extraMeta" passed (as saved)
                         ));
 
+												$this->events_manager->invoice()->created( array_merge( $data, array( 'id' => $newID ) ) ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
```

Comment:
Why not: `$data['id'] = $newID` before launching the event?

Reply:
I was just being extra careful not to change variables that could be used later, but `$data` is not used anywhere else, so I agree that it is safe to change it. Addressed in da42ac3.